lib/grape-swagger.rb

```diff
@@ -65,7 +65,6 @@ def combine_routes(app, doc_klass)
           next unless route_match
           resource = route_match.captures.first
           next if resource.empty?
-          resource.downcase!
```

Comment:
maybe you can re-use this method [operation_id.rb#L13](https://github.com/ruby-grape/grape-swagger/blob/master/lib/grape-swagger/doc_methods/operation_id.rb#L13)